kernel/classes/datatypes/ezuser/ezusertype.php

```diff
@@ -258,10 +258,22 @@ function storeObjectAttribute( $contentObjectAttribute )
             }
 
             // saving information in the object attribute data_text field to simulate a draft
-            $contentObjectAttribute->setAttribute( 'data_text', $this->serializeDraft( $user ) );
+            // only if the object version is a draft (status == 0)
+            if(
+                $user->Login &&
+                $contentObjectAttribute->attribute( 'object_version' )->attribute( 'status' ) == 0
```

Comment:
On the other hand, it makes sense here to use `eZContentObjectVersion::STATUS_DRAFT` instead of just `0`. Same in the comment above it.

Reply:
The comment could even be removed if we use the constant, since that makes the intention clear.

Reply:
That's a good point. I changed it now.